Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/CWFS.rst

```diff
@@ -4,11 +4,94 @@
+- The sky brightness is low enough that a CWFS can converge succesfully. This condition is normally achieved around 10-15 minutes before the end of nautical twilight or dark time. 
+
+.. _Beginning_of_the_night-Post-Conditions:
+
+Post-Condition
+===============
+
+- Telescope is focused and pointing offsets have been absorbed. The system is ready to start scheduler-driven observations (:ref:`Scheduler <Nighttime-Scheduler-Scheduler-Operational-Procedures>`) or engineering tasks. 
+
+.. _Beginning_of_the_night-Procedure-Steps:
+
+Procedure Steps
+===============
+
+- Confirm sky background is dark enough. 
+
+As a general rule, a good time to start executing the following SAL scripts is 10-15 minutes before the beginning of dark time. 
+
+- Launch a CWFS sequence on an 8th magnitude star. 
+
+From the LOVE ``ATQueue`` panel, under ``AVAILABLE SCRIPTS`` on the left side, find and load the script ``auxtel/latiss_cwfs_align.py`` by clicking on the blue icon. 
+It is included under ``External scripts``. 
+
+In the ``configuring Script: latiss_cwfs_align`` window, the script configuration is introduced under ``CONFIG``.
+Traditionally, we perform the first CWFS of the night towards the south and at mid-elevations on an 8-mag star. 
+With the configuration below, the script will query SIMBAD for a target around the provided ``az`` and ``el`` coordinates and with the specified maximum brightness ``mag_limit``.
+
+.. code-block:: text
+    
+    find_target:
+        az: 180
+        el: 60
+        mag_limit: 8.0
+    
+Click on ``Add`` and the script will be added to the end of the script queue.
+
+.. figure:: ./_static/CWFS_AuxTel.png
+    :name: latiss-cwfs
+
+    Screenshot of LOVE launching a CWFS. 
+
+- Instruct the telescope to correct the pointing, if needed. 
+
+This SAL script will measure and apply the zero point correction to the telescope pointing. 
+The steps the script perform in the background are:
+    - Reset the pointing and hexapod x and y offsets.
+    - Acquire target. How to select the source is summarize below. 
+    - Find the offset between the brightest source in the field and the boresight.
+    - Offset the telescope to place the source at the center of the detector. 
+    - The telescope will absorb the measured pointing offsets. 
+
+Nominally, the same target operated in the CWFS is used.  
+Add the external script ``auxtel/correct_pointing.py`` to the ``AT Queue`` with an empty configuration cell. 
+
+.. figure:: ./_static/CorrectPointing_AuxTel.png
+    :name: correctPointing-AuxTel
+
+    LOVE launching the script to correct the pointing. 
+
+To choose a different target, the ``find_target`` configuration is required to select a source with the given coordinates and magnitude limit.
+Add the external script ``auxtel/correct_pointing.py`` to the ``AT Queue`` with the ``find_target`` configuration, similarly to the CWFS:
+
+.. code-block:: text
+    
+    find_target:
+        az: 90
```

Comment:
So, this configuration and the one proposed for was are different. We should probably use the same.

Reply:
Do you mean the CWFS configuration above? I have matched both to AZ 180 and EL 60.